webapp/channels/src/components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -105,10 +104,20 @@ class AnnouncementBarController extends React.PureComponent<Props> {
             );
         }
 
+        let usersLimitsAnnouncementBar = null;
+        if (this.props.userIsAdmin) {
```

Comment:
Should we move this condition within the user limit banner to encapsulate all conditions of when to show there and be able to test them in within the `shouldShowUserLimitsAnnouncementBar` you extracted?

Reply:
If we do that, we'd be checking for CWS availability for all users, which would unnecessarily add inbound traffic on cws API.

Reply:
> If we do that, we'd be checking for CWS availability for all users, which would unnecessarily add inbound traffic on cws API.

True but we could also move that condition to check for admin before trying to check for airgapped. 0/5 - just thinking that all business logic for when to show the banners seems to be kept within each banner

Reply:
We currently do that for in the signup for all users anyways. I agree all logic inside the component is good for scope.